client/ayon_core/hosts/nuke/plugins/load/load_clip.py

```diff
@@ -130,6 +130,19 @@ def load(self, context, name, namespace, options):
             first = 1
             last = first + duration
 
+        # If a slate is present, the frame range is 1 frame longer for movies,
+        # but file sequences its the first frame that is 1 frame lower.
+        slate_frame = repre_entity["data"].get("slateFrame", False)
+        if slate_frame:
+            extension = "." + repre_entity["context"]["ext"]
+
+            if extension in VIDEO_EXTENSIONS:
+                last += 1
```

Comment:
This does not look right. How is it possible that video file is shorter one frame?

Reply:
It's not shorter? It's adding to the `last` (which is the `end`, right?)

So there's an extra frame at the end.
However, as the comment says - for image sequences instead there's an extra frame at the start.

Which admittedly the video example sounds wrong. The timecode for the slate frame should still be 'before' the start of the video.